app/parse.py

```diff
@@ -8,5 +13,24 @@
     duration: str
 
 
+def get_number_from_months_text(text: str) -> str:
+    return text.split(" ")[0][:-1]
+
+
+def parse_single_course(course: Tag) -> Course:
+    name = course.select_one("a h3").text
```

Comment:
The code assumes that the 'a h3' selector will always return a valid element with text. If the structure of the HTML changes, this could lead to an `AttributeError`. Consider adding a check to ensure the element exists before accessing its text.